### ISSUE TITLE
graphql-code-generator broken with graphql-js 0.13.0

### ISSUE THREAD

**Issue by jonaskello** [dotansimha/graphql-code-generator#184]
https://github.com/graphql/graphql-js/issues/1221

I need to use 0.13.0 becuase there is a bug in 0.12.x that makes minification impossible. And all packages need the same version of graphql-js to make the instanceof checks work. And graphql-code-generator does not work with ^0.13.0 becuase it relies on `'graphql/language/kinds'`.

Could it please be updated to work with 0.13.0?

**Comment by GD-Aichi** [comment]
transform-document.ts
```diff
- import { OPERATION_DEFINITION, FRAGMENT_DEFINITION } from 'graphql/language/kinds';
+ import { Kind } from 'graphql';
...
- if (definitionNode.kind === OPERATION_DEFINITION) {
+ if (definitionNode.kind === Kind.OPERATION_DEFINITION) {
...
- } else if (definitionNode.kind === FRAGMENT_DEFINITION) {
+ } else if (definitionNode.kind === Kind.FRAGMENT_DEFINITION) {
```
build-selection-set.ts
```diff
- import { FIELD, FRAGMENT_SPREAD, INLINE_FRAGMENT } from 'graphql/language/kinds';
+ import { Kind } from 'graphql';
...
- if (selectionNode.kind === FIELD) {
+ if (selectionNode.kind === Kind.FIELD) {
...
- } else if (selectionNode.kind === FRAGMENT_SPREAD) {
+ } else if (selectionNode.kind === Kind.FRAGMENT_SPREAD) {
...
- } else if (selectionNode.kind === INLINE_FRAGMENT) {
+ } else if (selectionNode.kind === Kind.INLINE_FRAGMENT) {
```
this is the update

**Comment by dotansimha** [comment]
WIP: https://github.com/dotansimha/graphql-code-generator/pull/187

**Comment by jonaskello** [comment]
Looks awsome :-)

**Comment by dotansimha** [comment]
Fixed in 0.8.15 :) @jonaskello  @GD-Aichi
